web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -178,11 +97,6 @@ export const DailyViewTable = (props: Props): JSX.Element => {
 
   return (
     <FireContainer testId={props.testId}>
-      <div className={classes.controls}>
-        <Typography component="div" variant="subtitle2">
-          {props.title}
-        </Typography>
-      </div>
       <FireTable
         maxHeight={700}
```

Comment:
Why are we limiting to 700 pixels? I have a huge screen, and everything is squished to the top. (I guess this is an existing limitation?) I've added a ticket for it.

Reply:
It's because the headers are not sticky if it takes up the full screen.